beacon_chain/state/helpers.py

```diff
@@ -1,72 +1,129 @@
 from beacon_chain.state.config import (
     DEFAULT_CONFIG,
 )
+from beacon_chain.state.shard_and_committee import (
+    ShardAndCommittee,
+)
+from beacon_chain.utils.blake import (
+    blake,
+)
+
+
+def is_power_of_two(num):
+    return ((num & (num - 1)) == 0) and num != 0
+
+
+def get_parent_hashes(active_state,
```

Comment:
IMO it's not very clear that `get_parent_hashes` actually includes `oblique_parent_hashes`. What do you think about changing it to `get_signed_parent_hashes` or other names?